task3/task3.test.ts

```diff
@@ -12,5 +12,32 @@ describe('task3', () => {
     mockDate.reset();
   });
 
-  // write your tests here
+  it('returns correct date without argument passed', () => {
+    mockDate.set({ isoDate: '2019-03-05T15:00:00.000Z' });
+
+    expect(getUtcStringDate()).toBe('2019-03-05T15:00:00.000Z');
```

Comment:
It should be without milliseconds:
```suggestion
    expect(getUtcStringDate()).toBe('2019-03-05T15:00:00Z');
```